scipy/interpolate/tests/test_rgi.py

```diff
@@ -17,6 +17,10 @@
 parametrize_rgi_interp_methods = pytest.mark.parametrize(
     "method", ['linear', 'nearest', 'slinear', 'cubic', 'quintic', 'pchip']
 )
+parametrize_interpolators = pytest.mark.parametrize(
+    "interpolator", [NearestNDInterpolator, LinearNDInterpolator,
+                     CloughTocher2DInterpolator]
+)
```

Comment:
Ugh, what are these doing here? `test_readonly` should be moved to wherever other `griddata`  implementation is tested, `test_interpnd.py` or `test_ndgriddata.py`. Could you do it in a separate PR which we'll merge quickly @j-bowhay ?

Reply:
Yes these do seem out of place, will open a pr

Reply:
@ev-br the test for `NearestNDInterpolator`, `LinearNDInterpolator`, `CloughTocher2DInterpolator` are split over two files:

- `NearestNDInterpolator` is tested in `test_ndgriddata.py`
- `LinearNDInterpolator`, `CloughTocher2DInterpolator` are tested in `test_interpnd.py`

So can this test go in a common file that tests all 3 (if so which one would be best?) or does in need to go in both?

Reply:
Not that it matters all that much. I'd put them into test_ndgriddata.py, but if you've an opinion, expect no argument from me :-).